src/lando/main/models/repo.py

```diff
@@ -96,10 +139,12 @@ def get_mapping(cls) -> dict[str, "Repo"]:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.path = self.system_path or self.get_system_path()
```

Comment:
The fact that there is both `self.system_path` and `self.get_system_path()` is a bit odd, yeah. The getter constructs a new string, and is used in the `save()` method if `self.system_path` is not defined. I suspect this is needed to initialise new objects the first time round.

Conversely, the `path` seems to be there to hide this ambivalence, so I'm not sure we can remove it. I tried, and it upset a few tests, which were easily fixed by dependings on the system_path instead, but it doesn't seem like the right way to go. I think it would make sense to make it a property 👍🏼